roles/ceph-monitor/tasks/monitoring.yml

```diff
@@ -1,7 +1,13 @@
 ---
 - name: ceph status check
   sensu_check: name=check-ceph-status plugin=check-ceph.rb
-               use_sudo=true args='--criticality {{ ceph.monitoring.sensu_checks.check_ceph_status.criticality }}'
+               use_sudo=true args='--criticality {{ ceph.monitoring.sensu_checks.check_ceph_status.criticality }} -e "osds are down,near full osd,clock skew,mons down" -o -f -p'
```

Comment:
one question: if add this variable defined in the role default, then all existing blue box env file should be modified too